Project_0.Altis/CivilianPresence/fn_spawn.sqf

```diff
@@ -0,0 +1,60 @@
+
```

Comment:
Maybe turn it into a class some time later? For now it will do.

Reply:
Yes i thought about merging it with location and then make the functions in sepperate files